src/main/java/frc/robot/Constants.java

```diff
@@ -49,6 +49,9 @@
 
         public static final double STOP_CLIMBER = 149.5; // [s]
 
+        public static final double SIMULATION_LOOP_PERIOD = 0.02; // [s]
```

Comment:
Isn't there already a "loop period" constant?

Reply:
not for the simulations